extension-manifest-v3/src/store/options.js

```diff
@@ -76,21 +119,20 @@ const Options = {
       });
 
       // Send update message to another contexts (background page / panel / options)
-      try {
-        chrome.runtime.sendMessage({
-          action: UPDATE_OPTIONS_ACTION_NAME,
-          options,
-        });
-      } catch (e) {
-        console.error(
-          `Error while sending update options to other contexts: `,
-          e,
-        );
-      }
+      chrome.runtime
+        .sendMessage({ action: UPDATE_OPTIONS_ACTION_NAME })
+        .catch(() => {});
```

Comment:
please add a log

Reply:
The function only `throws` if the other end does not exist. Mainly, when the background process starts, but there is another content script or extension page, which could receive a message. This error is annoying (as the behavior is expected). The log message would introduce again something that I wanted to avoid.

Reply:
Got it. Please leave a comment with an explanation